internal/compiler/passes/const_propagation.rs

```diff
@@ -199,6 +254,30 @@ fn extract_constant_property_reference(nr: &NamedReference) -> Option<Expression
     Some(expression)
 }
 
+fn try_inline_function(function: &Callable, arguments: &[Expression]) -> Option<Expression> {
+    let Callable::Function(function) = function else {
```

Comment:
Note for future improvements: we could also try to inline some built-in functions.